guides/slack-guide.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: 'How to use Novu to send notifications to a Slack channel'
+description: 'Leverage the integration store to send Slack notifications'
```

Comment:
This description needs some change. Maybe something like:

"Learn to send Slack notifications swiftly with Novu"